apps/arena/lib/physics.ex

```diff
@@ -39,4 +39,6 @@ defmodule Physics do
   def distance_between_entities(_entity, _entities), do: :erlang.nif_error(:nif_not_loaded)
 
   def nearest_entity_position_in_range(_entity, _entities, _range), do: :erlang.nif_error(:nif_not_loaded)
+
+  def process_polygon_triangulation(_obstacles), do: :erlang.nif_error(:nif_not_loaded)
```

Comment:
I think `triangulate_concave_entities` is a more descriptive name.

Reply:
yeah i think it's a better name, i'll add a "maybe_" since it doesn't always triangulate